core/src/main/resources/SqlTranslationPatterns.properties

```diff
@@ -12,18 +12,20 @@
 
 # Oracle source used a.o. in testscenarios: ApiListener, JdbcQueryListener, BatchFileTransformerPipe, (Big)ClobAndBlob, XmlQuerySender
 # 'nextval' as the value to insert in the autoincrement column
-nextval.source.Oracle  =(\w+)\.NEXTVAL
-nextval.target.MS_SQL  =NEXT VALUE FOR $1
-nextval.target.MySQL   =NULL
-nextval.target.MariaDB =NULL
-nextval.target.PostgreSQL =nextval('$1')
+nextval.source.Oracle		=(\w+)\.NEXTVAL
+nextval.target.MS_SQL		=NEXT VALUE FOR $1
+nextval.target.MySQL		=NULL
+nextval.target.MariaDB		=NULL
+nextval.target.PostgreSQL	=nextval('$1')
+nextval.target.H2			=NEXT VALUE FOR $1
```

Comment:
Could you please use the same order of targets in each group, to improve readability?